src/SDKs/MixedReality/MixedReality.Tests/MixedReality.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>MixedReality.Tests</PackageId>
+    <Description>MixedReality.Tests Class Library</Description>
+    <AssemblyName>MixedReality.Tests</AssemblyName>
+    <VersionPrefix>0.9.0-preview</VersionPrefix>    
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
```

Comment:
@XiangyuL-Microsoft please remove the reference to Resource Manager
This reference is implicitly added to every test project.
Adding this separately will hamper our ability to update common references quickly